pkg/syncer/cnsoperator/controller/cnsnodevmattachment/cnsnodevmattachment_controller.go

```diff
@@ -260,7 +260,7 @@ func (r *ReconcileCnsNodeVMAttachment) Reconcile(ctx context.Context,
 				log.Errorf("updateCnsNodeVMAttachment failed. err: %v", err)
 			}
 			recordEvent(ctx, r, instance, v1.EventTypeWarning, msg)
-			return reconcile.Result{RequeueAfter: timeout}, "NotComputed", nil
+			return reconcile.Result{RequeueAfter: timeout}, csifault.CSIApiServerOperationFault, nil
```

Comment:
Wouldn't this be VCenterConnectionFault or an internal fault?

Reply:
Yes, should be internal fault